pkg/kubelet/dockertools/docker.go

```diff
@@ -276,16 +277,19 @@ func LogSymlink(containerLogsDir, podFullName, containerName, dockerId string) s
 	return path.Join(containerLogsDir, fmt.Sprintf("%s_%s-%s.%s", podFullName, containerName, dockerId, LogSuffix))
 }
 
-// Get a *docker.Client, either using the endpoint passed in, or using
+// Get a *dockerapi.Client, either using the endpoint passed in, or using
 // DOCKER_HOST, DOCKER_TLS_VERIFY, and DOCKER_CERT path per their spec
-func getDockerClient(dockerEndpoint string) (*docker.Client, error) {
+func getDockerClient(dockerEndpoint string) (*dockerapi.Client, error) {
 	if len(dockerEndpoint) > 0 {
 		glog.Infof("Connecting to docker on %s", dockerEndpoint)
-		return docker.NewClient(dockerEndpoint)
+		return dockerapi.NewClient(dockerEndpoint, "", nil, nil)
 	}
-	return docker.NewClientFromEnv()
+	return dockerapi.NewEnvClient()
 }
 
+// ConnectToDockerOrDie creates docker client connecting to docker daemon.
+// If the endpoint passed in has the prefix "fake://", a fake docker client
```

Comment:
Yeah...Maybe we could remove it in the future.
BTW, mark here: `s/has the prefix/is`